### PR TITLE
[BLOG] Add numpy_CZI_grant

### PR DESCRIPTION
**Pull request by MarsBarLee**
Link to [blog post, successful build](https://labs-git-blog-name-quansight.vercel.app/blog/blog-name).

## Links to other Quansight blogs:
There is no need to check for Blog2Blog links due to updated website build. 
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

File: apps/labs/posts/numpy_CZI_grant.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+---
+title: 'A new grant for NumPy and OpenBLAS!'
+published: November 14, 2019
+author: ralf-gommers
+description: 'I'm very pleased to announce that NumPy and OpenBLAS just received a $195,000 grant from
+the Chan Zuckerberg Initiative, through its Essential Open Source Software for Science
+(EOSS) program! '
+category: [Funding, PyData Ecosystem]
```
Comment (MarsBarLee) [inline apps/labs/posts/numpy_CZI_grant.md:8]:
```suggestion
category: [Funding, PyData ecosystem]
```

File: apps/labs/posts/numpy_CZI_grant.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+featuredImage:
+  src: /posts/blog_feature_var2.svg
```
Comment (MarsBarLee) [inline apps/labs/posts/numpy_CZI_grant.md:10]:
```suggestion
  src: /posts/numpy_CZI_grant/blog_feature_var2.svg
```

File: apps/labs/posts/numpy_CZI_grant.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+  alt: 'An illustration of a brown and a white hand coming towards each other to pass a business card with the logo of Quansight Labs.'
+hero:
+  imageSrc: /posts/blog_hero_var2.svg
```
Comment (MarsBarLee) [inline apps/labs/posts/numpy_CZI_grant.md:13]:
```suggestion
  imageSrc: /posts/numpy_CZI_grant/blog_hero_var2.svg
```

**Review by noatamir — APPROVED** [review]
LGTM

**Review by rgommers — APPROVED** [review]
LGTM. I'm out of the loop on website infra, so I'll let someone else merge this.

**Review by trallard — APPROVED** [review]
LGTM - merging now!

**Pull request merged**